tick/solver/scpg.py

```diff
@@ -230,6 +233,7 @@ def set_model(self, model: ModelSecondOrder):
         self._set('model', model)
         self._set('model_ssc', self._ModelStandardSC(model=model))
         self.prox_ssc.set_self_conc_constant(model._sc_constant)
+        self.dtype = np.dtype("float64")
```

Comment:
Why don't we set self.dtype to self.model.dtype ? (would not be a problem if dtype becomes a model property)

Reply:
SCPG has no templates so is always float64